src/test/java/teammates/test/util/Url.java

```diff
@@ -0,0 +1,16 @@
+package teammates.test.util;
+
+import teammates.test.driver.TestProperties;
+
+public class Url extends teammates.common.util.Url {
```

Comment:
I'll merge this and work on all the issues (renaming `Url`, header comments, ...) in one PR right away. How about it?